test/expected/bgw_db_scheduler.out

```diff
@@ -573,6 +573,83 @@ SELECT * FROM sorted_bgw_log;
       1 |    800000 | test_job_3_long  | After sleep job 3
 (11 rows)
 
+--Test sending a SIGHUP to a job
+\c :TEST_DBNAME :ROLE_SUPERUSER
+TRUNCATE bgw_log;
+SELECT ts_bgw_params_reset_time();
+ ts_bgw_params_reset_time 
+--------------------------
+ 
+(1 row)
+
+TRUNCATE _timescaledb_internal.bgw_job_stat;
+DELETE FROM _timescaledb_config.bgw_job;
+\c :TEST_DBNAME :ROLE_DEFAULT_PERM_USER
+--escalated priv needed for access to pg_stat_activity
+\c :TEST_DBNAME :ROLE_SUPERUSER
+SELECT ts_bgw_db_scheduler_test_run(-1);
+ ts_bgw_db_scheduler_test_run 
+------------------------------
+ 
+(1 row)
+
+SHOW timescaledb.shutdown_bgw_scheduler;
+ timescaledb.shutdown_bgw_scheduler 
+------------------------------------
+ off
+(1 row)
+
+ALTER SYSTEM SET timescaledb.shutdown_bgw_scheduler TO 'on';
+SELECT pg_reload_conf();
+ pg_reload_conf 
+----------------
+ t
+(1 row)
+
+SELECT pg_sleep(0.001);
```

Comment:
why is this sleep necesssary? shouldn't ts_bgw_db_scheduler_test_wait_for_scheduler_finish(); wait correctly?

Reply:
@cevian It takes time for the SIGHUP to be handled, putting the process to sleep, however briefly, guarantees that the process will handle the SIGHUP, and reload the config file. I want the SHOW to work, and show the new version of the guc, because that'll be the first thing we want to check when this has bugs ^__^